src/olcs/contrib/manager.js

```diff
@@ -44,6 +45,12 @@ olcs.contrib.Manager = class {
      * @protected
      */
     this.ol3d;
+
+    /**
+     * @const {number} Tilt angle in radians
+     * @private
+     */
+    this.cesiumInitialTilt_ = olcs.core.toRadians(50);
```

Comment:
Use `Cesium.Math.toRadians` (or `ol.math.toRadians`) instead

Reply:
Thanks for the suggestion. I updated the PR.